nav2_waypoint_follower/src/waypoint_follower.cpp

```diff
@@ -33,6 +33,8 @@ WaypointFollower::WaypointFollower()
 
   declare_parameter("stop_on_failure", true);
   declare_parameter("loop_rate", 20);
+  declare_parameter("gps_waypoint_transform_timeout", 0.1);
```

Comment:
`transform_tolerance` is the standard name for this param that all the other servers use, we should make sure to be consistent